spec/lib/active_merchant/billing/gateways/shopify_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'spec_helper'
+require 'active_merchant/billing/gateways/shopify'
+
+module ActiveMerchant::Billing
+  RSpec.describe ShopifyGateway do
+    let(:api_key) { 'api_key' }
+    let(:password) { 'password' }
+    let(:shop_name) { 'shop_name' }
+
+    let(:gateway) do
+      described_class.new(api_key: api_key,
+                          password: password,
+                          shop_name: shop_name)
+    end
+
+    around(:each) do |example|
+      begin
+        previously = ShopifyAPI::Base.site
+        example.run
+      ensure
+        ShopifyAPI::Base.site = previously
+      end
+    end
+
+    describe "#void" do
```

Comment:
Can this be dramatically pared down because these behaviours have been delegated to `ShopifyVoider`?